### DISCUSSION TITLE
logging into container from console

### DISCUSSION THREAD

**Discussion by kinghat** [lxdware/lxd-dashboard#22] [Q&A] [answered]
im pretty new to lxc containers and i know the images are usually setup with root only and no password but am confused as to what should be used to login to a container from the web console? i really appreciate you documenting other possible pain points that people me run across on your website.

**Comment by matthewalanpenning** [comment, marked as answer]
That is a great question. The web console provides an interface similar to what you would expect when connecting a monitor to your instance. For all of the instances that I have worked that are based on linuxcontainers.org and ubuntu.com images there are no accounts setup with a password to login. So in order to use the console you will either need to set a password for the root user or create a new user account in the container first. This will then allow you to login via the console.
